assets/js/src/core/modules/element/hooks/use-element-draft.ts

```diff
@@ -20,15 +20,15 @@ import type {
 } from '@Pimcore/modules/element/draft/hooks/use-trackable-changes'
 import { useAssetDraft } from '@Pimcore/modules/asset/hooks/use-asset-draft'
 import { useDataObjectDraft } from '@Pimcore/modules/data-object/hooks/use-data-object-draft'
-import {
-  type ElementEditorType
-} from '@Pimcore/modules/element/editor/services/type-registry'
+import { type ElementEditorType } from '@Pimcore/modules/element/editor/services/type-registry'
+import { type Permissions } from '@Pimcore/modules/asset/asset-api-slice.gen'
```

Comment:
Same as above - here we are in the element namespace but only support the asset permissions.

Reply:
good point. Where can I create a dedicated interface for `ElementPermissions`. Since it makes no sense to write `DataObjectPermissions | AssetPermissions`

Where would you put it?

Reply:
fixed with: https://github.com/pimcore/studio-ui-bundle/pull/692#discussion_r1842329625